### DISCUSSION TITLE
Edge caching query strings & request bodies?

### DISCUSSION THREAD

**Discussion by rayshan** [vercel/vercel#4938] [Help] [answered]
Hi! Would the serverless functions edge cache cache these cases?

1. GET request with query strings: `domain.com/api/?cars=[a,b,c]`
2. GET request with body payload: `domain.com/api/cars`

```json
["a", "b", "c"]
```

Couldn't find this info in the doc here: https://vercel.com/docs/v2/serverless-functions/edge-caching

**Comment by juancampa** [comment, marked as answer]
Hi @rayshan,

1. Yes! As long as the response is [cacheable](https://vercel.com/docs/v2/edge-network/caching#cacheable-responses)
2. The request body doesn't affect the cache at all so we strongly suggest you use the first option. In this case the response will be cached for `domain.com/api/cars` and the Edge will respond with it regardless of the body of any subsequent requests.

Hope this helps.

**Reply by rayshan** [reply]
Perfect thank you!
